tests/e2e/raw_block_volume.go

```diff
@@ -48,23 +54,32 @@ const (
 
 var _ = ginkgo.Describe("raw block volume support", func() {
 
-	f := framework.NewDefaultFramework("e2e-vsphere-statefulset")
+	f := framework.NewDefaultFramework("e2e-raw-block-volume")
 	f.NamespacePodSecurityEnforceLevel = admissionapi.LevelPrivileged
 	var (
-		namespace           string
-		client              clientset.Interface
-		defaultDatacenter   *object.Datacenter
-		datastoreURL        string
-		scParameters        map[string]string
-		storageClassName    string
-		storagePolicyName   string
-		svcPVCName          string
-		rawBlockVolumeMode  = corev1.PersistentVolumeBlock
-		pandoraSyncWaitTime int
-		deleteFCDRequired   bool
-		fcdID               string
-		pv                  *corev1.PersistentVolume
-		pvc                 *corev1.PersistentVolumeClaim
+		namespace                 string
```

Comment:
can you think of adding vcp-csi test to a new file or to existing file with vcp tests

Reply:
done. moved the testcase to existing tc vcp_to_csi_syncer.go